### PR TITLE
Add support for non-standard MPRIS ratingsExtension

### PR DESCRIPTION
**Pull request by JasonLG1979**
This will allow any MPRIS client to set ratings in Lollypop via the SetRating Method. This is also future proof in the sense that if/when the TrackList interface is implemented it will allow clients to set the ratings of any song in the tracklist. Support for this interface has already been added to https://github.com/JasonLG1979/gnome-shell-extensions-mediaplayer.

### CODE REVIEW

File: lollypop/mpris.py [hunk 1]
```diff
@@ -310,6 +323,9 @@ def GetAll(self, interface):
                                   "CanSeek",
                                   "CanControl"]:
                 ret[property_name] = self.Get(interface, property_name)
+        elif interface == self.__MPRIS_RATINGS_IFACE:
+            for property_name in ["HasRatingsExtension"]:
```
Comment (bil-elmoussaoui) [inline lollypop/mpris.py]: Il's not needed at all... there's only one element in the list

Comment (JasonLG1979) [inline lollypop/mpris.py]: I''ll fix it good catch. I was basically just copy pasta.

Comment (JasonLG1979) [inline lollypop/mpris.py]: Done.

**Comment by JasonLG1979** [comment]
This also has the benefit of being much faster then setting ratings via the command line. As in there is less lag between when a user rates a track via my extension and when Lollypop emits a metadata prop change signal.

**Comment by gnumdk** [comment]
Thanks

**Comment by JasonLG1979** [comment]
I'd like to implement the PlayList interface if you'd point me in the right direction as far as how to get a the playlists in Lollypop?

**Pull request merged**